apps/e2e/forms-storybook/src/app/checkbox/checkbox.component.html

```diff
@@ -251,4 +272,23 @@
       </sky-checkbox>
     </sky-column>
   </sky-row>
+  <sky-row class="sky-padding-even-md">
+    <sky-column>
+      <form [formGroup]="iconCheckboxGroupFormGroup">
+        <sky-checkbox-group
+          [formGroup]="iconCheckboxGroupFormGroup"
+          labelText="Text formatting"
+          [labelHidden]="true"
```

Comment:
Are we intentionally hiding the label for the icon group?

Reply:
My recollection is that we decided against that due to uncertainty around whether or not consumers would need it and the fact that detecting icon checkboxes would be a bit ick. Paul agrees with making them hide it, however he doesn't remember the conversation 😂 happy to bring it up in flex tomorrow if you feel it should be automatic.

Reply:
I just meant - did we mean to hide it for the screenshots - sounds like yes? I just noticed that we didn't have a screenshot with both icon checkboxes and the label.

Reply:
oh, yes I meant to do that. There is one checkbox group with a label shown and one with a label hidden.